docs/cmd/kumactl/HELP.md

```diff
@@ -189,7 +189,8 @@ Usage:
   kumactl install [command]
 
 Available Commands:
-  control-plane Install Kuma Control Plane on Kubernetes
+  control-plane   Install Kuma Control Plane on Kubernetes
+  database-schema Install Kuma schema on DB.
```

Comment:
It would be great if command descriptions were consistent, e.g. in terms of a trailing full stop in the end of a sentence